airgun/widgets.py

```diff
@@ -1,32 +1,32 @@
 from cached_property import cached_property
 from wait_for import wait_for
-from widgetastic.exceptions import NoSuchElementException
-from widgetastic.exceptions import WidgetOperationFailed
-from widgetastic.widget import Checkbox
-from widgetastic.widget import ClickableMixin
-from widgetastic.widget import do_not_read_this_widget
-from widgetastic.widget import GenericLocatorWidget
-from widgetastic.widget import ParametrizedLocator
-from widgetastic.widget import Select
-from widgetastic.widget import Table
-from widgetastic.widget import Text
-from widgetastic.widget import TextInput
-from widgetastic.widget import View
-from widgetastic.widget import Widget
+from widgetastic.exceptions import NoSuchElementException, WidgetOperationFailed
+from widgetastic.widget import (
+    Checkbox,
+    ClickableMixin,
+    GenericLocatorWidget,
+    OUIAContextSelector,
```

Comment:
@JacobCallahan This should be coming from `widgetastic_patternfly4.ouia` on line number 27, PRT failing because of this

Reply:
great catch!